src/interface/desktop/main.js

```diff
@@ -575,9 +608,39 @@ app.whenReady().then(() => {
             console.warn("Desktop app update check failed:", e);
         }
     })
-
+    openShortcut = false;
+    globalShortcut.register(openShortcutWindowKeyBind, () => {
+        if(openShortcut) return;
+        const shortcutWin = createShortcutWindow(); // Create a new shortcut window each time the shortcut is triggered
+        shortcutWin.setAlwaysOnTop(true, 'screen-saver', 1);
+        const clipboardText = clipboard.readText();
+        console.log('Sending clipboard text:', clipboardText); // Debug log
+        shortcutWin.webContents.once('dom-ready', () => {
+          shortcutWin.webContents.send('clip', clipboardText);
+          console.log('Message sent to window'); // Debug log
+        });
+        openShortcut = true;
+        // Register a global shortcut for the Escape key for the shortcutWin
+        globalShortcut.register('Escape', () => {
+          if (shortcutWin) {
+            shortcutWin.close();
+          }
+          // Unregister the Escape key shortcut
+          globalShortcut.unregister('Escape');
+          openShortcut = false;
+        });
+        ipcMain.on('button-clicked', () => {
```

Comment:
Let's give this a more descriptive name to avoid collisions in the future? Like `continue-conversation-button-clicked`?